frontend/containers/forms/filters/component.tsx

```diff
@@ -110,80 +116,148 @@ export const Filters: FC<FiltersProps> = ({
+                        <span className="mr-2 font-sans text-sm font-semibold text-gray-800">
+                          {filter.title}
+                        </span>
+                        <FieldInfo
+                          content={
+                            filter.description ? (
+                              filter.description
+                            ) : (
+                              <ul>
+                                {filter.values?.map((value) => (
+                                  <li key={value.id} className="mb-2">
+                                    <p className="font-bold ">{value.name}</p>
+                                    {!!value.description && <p>{value.description}</p>}
+                                  </li>
+                                ))}
+                              </ul>
+                            )
+                          }
+                        />
+                      </legend>
 
-        <div className="flex flex-col flex-wrap gap-y-4">
-          {filters?.map((filter) => {
-            return (
-              <div key={filter.title}>
+                      {!!filter.values?.length && (
+                        <TagGroup
+                          name={filter.values?.[0].type}
+                          type="radio"
+                          clearErrors={clearErrors}
+                          setValue={setValue}
+                          errors={errors}
+                          thresholdToShowSelectAll={Infinity}
+                          className="flex flex-wrap gap-2"
+                          isFilterTag
+                        >
+                          {filter.values.map(({ name, id, type, description }) => (
+                            <Tag
+                              key={id}
+                              id={`${id}-tag`}
+                              name={type}
+                              value={id}
+                              register={register}
+                              registerOptions={{
+                                disabled: false,
+                                onChange,
+                              }}
+                              type="radio"
+                              isfilterTag
+                              onClick={onChange}
+                            >
+                              <span className="block">
+                                {type === EnumTypes.TicketSize ? description : name}
+                              </span>
+                            </Tag>
+                          ))}
+                        </TagGroup>
+                      )}
+                    </fieldset>
+                  </div>
+                );
+              })}
+            </div>
+            <div className="flex my-4">
+              {/* More filters accordion header https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion.html */}
+              <h3>
+                <Button
```

Comment:
The focus ring of the button is also cut on the left:
![The accordion's focus ring is cut on the left](https://user-images.githubusercontent.com/6073968/190176233-b70f7f9d-8b46-4a05-814c-278623b70c7f.png)